lib/guardian/channel.ex

```diff
@@ -37,14 +37,13 @@ defmodule Guardian.Channel do
   directly and authenticating the connection rather than the channel.
   """
   defmacro __using__(opts) do
-    opts = Enum.into(opts, %{})
-    key = Map.get(opts, :key, :default)
+    key = Keyword.get(opts, :key, :default)
```

Comment:
Warning:

```shell
lib/guardian/channel.ex:41: warning: variable key is unused
```

@hassox this was probably a bug, key wasn't being unquoted.  Can you review this change?